test/test_scatter_gather_ops.py

```diff
@@ -84,10 +84,19 @@ def test_gather_large(self, device, dtype):
                 res = torch.gather(src, dim=dim, index=ind)
                 ref = src[ind0] if dim == 0 else src[:, ind0]
                 self.assertEqual(res, ref, atol=0, rtol=0)
+                if res.device.type == "cuda":
```

Comment:
Should this be if '== "cuda"' or really if '!= "cpu"'?

Reply:
yes, this is specifically for cuda implementation